src/components/UserFeedbackComponent.jsx

```diff
@@ -115,7 +115,7 @@ const mapStateToProps = ({ feedbackReducer }) => {
   return {
     feedback,
     feedbackCount,
-    hasFeedback: !!feedbackCount,
+    hasFeedback: !feedbackCount,
```

Comment:
@3m3kalionel the double exclamation mark is necessary as `!feedbackCount` will evaluate `feedbackCount > 0` to false, yet it should be true.  I have attached a console screenshot to illustrate what I mean
![image 2018-07-05 at 11 04 23 am](https://user-images.githubusercontent.com/12892171/42310112-39dfeb5c-8043-11e8-8103-0069b31186ed.png)